bazel/fuzz_target.bzl

```diff
@@ -62,13 +68,20 @@ def java_fuzz_target_test(
     else:
         fail("Invalid sanitizer: " + sanitizer)
 
+    if type(expected_finding) == type(""):
```

Comment:
If used, is it possible to evaluate `select` here and only use `string` afterwards? Could not find anything in the documentation on that. 
If so, we could use `string` values in `select` instead of arrays and move `expected_findings` in `args` a few lines below.

Reply:
Unfortunately that is not possible: `select` is a bit of magic in that it is not evaluated when called, but rather returns an opaque object that encodes the alternatives. This (currently) makes it impossible to unwrap select and manipulate its values, rather the only supported operation on it is list concatenation. That makes it difficult to improve the logic here.